Week06/lesson12/homework/rock_paper_scissors_starter/js/main.js

```diff
@@ -0,0 +1,27 @@
+// storing the running totals, starting at 0
+var humanScore = 0;
+var computerScore = 0;
```

Comment:
YES you want to store the running totals globally, and then you're going to want to update these variables as different players win. 👍